client/my-sites/plan-features/style.scss

```diff
@@ -195,6 +195,10 @@ $plan-features-header-banner-height: 20px;
 		border-bottom: none;
 	}
 
+	&:first-child {
+		border-top: solid 1px lighten( $gray, 27% );
```

Comment:
Nice.

I just noticed that the bottom border of the PlanFeaturesHeader is only `1px`, not `2px` as per Andy's design. Could you also update that, please?